notty-community.opam

```
@@ -1,17 +1,19 @@
-# This file is generated by dune, edit dune-project instead
 opam-version: "2.0"
 synopsis: "Declaring terminals"
-description:
-  "Notty is a declarative terminal library for OCaml structured around a notion of composable images. It tries to abstract away the basic terminal programming model, providing something simpler and more expressive."
+description: """
+Notty is a declarative terminal library for OCaml structured around a notion
+of composable images. It tries to abstract away the basic terminal
+programming model, providing something simpler and more expressive.
+"""
```

Do we need something like this?

```suggestion
Notty is a declarative terminal library for OCaml structured around a notion
of composable images. It tries to abstract away the basic terminal
programming model, providing something simpler and more expressive.

This is a branch from the original notty, maintained by the community.
"""
```